### DISCUSSION TITLE
Integrate LIghtHouse into an node.js appliation

### DISCUSSION THREAD

**Discussion by sajankota** [GoogleChrome/lighthouse#15267] [General] [answered]
Right now, I am running as an npm package in a nodejs application. I do not want to use Lighthouse as an npm package. I would want the lighthouse to be part of my node.js application. I would want to customize the look and feel of the reports generated. 

Is it possible to directly integrate lighthouse directly in a node js app?

How do I go about it? How can I integration lighthouse into a node.js application..Is this a good practice to include in a nodejs application?

Thanks
Thanks.

**Comment by adamraine** [comment, marked as answer]
Please have a look at our docs: https://github.com/GoogleChrome/lighthouse/tree/main/docs

If you want to customize the report, you can fork and modify our report code: https://github.com/GoogleChrome/lighthouse/tree/main/report

**Comment by sajankota** [comment]
Thank you @adamraine,  I was able to fork the repository and I was able to understand how to customize the reports.
